.golangci.yml

```diff
@@ -4,6 +4,7 @@ run:
     - ^.*\.(pb|y)\.go$
   skip-dirs:
     - "vendor$"
+    - "pkg/app/piped/executor/analysis/mannwhitney"
```

Comment:
According to #2501, this package is just a copy from [golang.org/x/perf/internal/stats](https://github.com/golang/perf/tree/master/internal/stats).

Some lint errors were detected(deadcode, ineffassign), but I decided to  ignore them because this package is not ours.